app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -1,35 +1,65 @@
 //= depend_on_asset "active_admin/datepicker/datepicker-input-icon.png"
-// -------------------------------------- Active Admin Forms
-form {
-  /* Reset margins & Padding */
-  ul, ol, li, fieldset, legend, input, textarea, select, p { margin:0; padding:0; }
-  ol, ul { list-style: none }
+
+input[type=text],
+input[type=password],
+input[type=email],
+input[type=number],
+input[type=url],
+input[type=tel],
+textarea {
+  border: $border-width solid #c9d0d6;
```

Comment:
Color literals like `#c9d0d6` should only be used in variable declarations; they should be referred to via variable everywhere else.